friends/src/components/Friends.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import axios from 'axios';
+
+
+class Friends extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            friends: [],
+            name: '',
+            age: 0,
+            email: '',
+        }
+    }
+
+    componentDidMount() {
+        axios
+          .get('http://localhost:5000/friends')
+          .then(response => this.setState({ friends: response.data }))
+          .catch(error => console.log(error));
+        //     this.setState({ items: data });
+      }
+
+    changeHandler = event => {
+    event.preventDefault();
```

Comment:
indent more, also you don't need an event.preventDefault in a changeHandler, there's no default behavior you need to prevent for updating an input element